provider/pkg/provider/local/commandController.go

```diff
@@ -81,6 +80,5 @@ func (c *Command) Delete(ctx p.Context, id string, props CommandOutputs) error {
 	if props.Delete == nil {
 		return nil
 	}
-	_, _, err := props.run(ctx, *props.Delete)
-	return err
+	return run(ctx, *props.Delete, &props.BaseInputs, &props.BaseOutputs)
```

Comment:
this is the bit that specifically fixes the linked issue right?

Reply:
No, the fix is provider/pkg/provider/local/runController.go:L31.